Minor linting problem

### Expected behaviour
ESlint returns no warnings or errors.

### Actual behaviour
ESlint reports an error: 
```
/Users/michal/work/lisk-explorer/src/components/market-watcher/market-watcher.service.js
  52:6  error  'System' is not defined  no-undef
```

### Steps to reproduce
``npm run start``

--- a/src/components/market-watcher/market-watcher.service.js
+++ b/src/components/market-watcher/market-watcher.service.js
@@ -1,3 +1,4 @@
+/* global System */
 import angular from 'angular';
 import AppMarketWatcher from './market-watcher.module';
 

